wger/nutrition/views/meal.py

```diff
@@ -25,6 +25,7 @@
 
 from wger.nutrition.models import NutritionPlan, Meal
 from wger.utils.generic_views import WgerFormMixin
+# from wger.nutrition.forms import MealForm
```

Comment:
please take this out if it isn't going to be used at all

Reply:
Thank you for the feedback. This has been done